app/partials/showModuleNewRelease.html

```diff
@@ -10,6 +10,8 @@
 <div class="row">
 
   <div class="jumbotron upload-jumbotron" ng-controller="ReleaseFileCtrl"
+  ui-view="upload_new_release"
```

Comment:
Out of interest, does `upload_new_release` mean anything? It's not actually a view id, right?